Add new servers

viveksingh101994 opened the pull request:
curl for testing
```
curl --request POST \
  --url http://localhost:3000/api/mcp-servers \
  --header 'Content-Type: application/json' \
  --data '
{
  "name": "github",
  "type": "streamable-http",
  "isDisabled": false,
  "url": "https://api.githubcopilot.com/mcp/",
  "headers": {
    "Authorization": "Bearer <personal_token>",
    "X-MCP-Toolsets": "all"
  }
}
'
```

ON UI, added streamable-http option (github works with it) other option was to use docker but that would be tricky to run inside the server

viveksingh101994 committed:
chore: adds mcp logic for github

TranHung92 commented on packages/server/src/mcp/client.ts:
```diff
@@ -63,6 +64,26 @@ class MCPClientManager {
+      }
+
+      // Build streamable HTTP transport options
+      const httpOpts: any = {};
```
better typing

viveksingh101994 replied:
resolves with https://github.com/reality-platforms/ebee-oss/pull/27/commits/82f42f5957f5fb2171de4c04b9e7d496bea5670f

TranHung92 commented on packages/server/src/services/sources/github/mcpClient.ts:
```diff
@@ -0,0 +1,1011 @@
+    let page = 1;
+    const perPage = 100;
+
+    while (true) {
```
this would never stop

viveksingh101994 replied:
resolves with https://github.com/reality-platforms/ebee-oss/pull/27/commits/51528f3fba8a8efaef9db7b250bc007c97015e8e

TranHung92 commented on packages/server/src/services/sources/github/mcpClient.ts:
there are a lot of unused methods. Let's clean those up. We can always add them back later on if needed

viveksingh101994 replied:
resolves with https://github.com/reality-platforms/ebee-oss/pull/27/commits/51528f3fba8a8efaef9db7b250bc007c97015e8e

TranHung92 commented on packages/server/src/services/sync/adapters/github-adapter.ts:
remove commented code if not needed

viveksingh101994 replied:
resolves with https://github.com/reality-platforms/ebee-oss/pull/27/commits/51528f3fba8a8efaef9db7b250bc007c97015e8e

viveksingh101994 committed:
remove and uncomment usable code

viveksingh101994 committed:
chore: adds type

viveksingh101994 committed:
Merge branch 'develop' of github.com:reality-platforms/ebee-oss into github_mcp

viveksingh101994 committed:
chore: adds fathom mcp (#31)

* chore: adds fathom mcp

* refactor(fathom-mcp-server): migrate to monorepo structure

Restructure fathom-mcp-server by moving it into the packages/ directory
to adopt a monorepo layout. Simplify npm scripts by consolidating dev
and start commands to directly execute TypeScript using tsx, eliminating
the need for a separate dev script and build step during local development.

Changes:
- Move all fathom-mcp-server files to packages/fathom-mcp-server/
- Remove separate 'dev' script from package.json
- Update 'start' script to use tsx for direct TypeScript execution
- Update pnpm-lock.yaml to reflect new package structure

* fixes for fathom

* chore: adds shared-util package

---------

Co-authored-by: awcchungster <awcchungster@gmail.com>
Co-authored-by: Alex C. <3067000+buildgreatthings@users.noreply.github.com>

viveksingh101994 committed:
chore: using pino logger

viveksingh101994 committed:
chore: fix build error

TranHung92 committed:
* feat(server): add Slack integration and expand source type support

**Changes:**
- Add @slack/web-api dependency for Slack API integration
- Implement SlackClient wrapper with rate limiting and data extraction methods
- Create SlackAdapter for syncing Slack channels, messages, and threads
- Expand source type enum to include fathom, whatsapp, codebase, asana, and google_drive
- Add parentId field to Record model for hierarchical data (threads, conversations)
- Integrate Slack adapter into index-graph queue processor

buildgreatthings commented on packages/server/src/services/sync/adapters/github-adapter.ts:
```diff
@@ -0,0 +1,1144 @@
+      // }
+
+      // // Fetch pull requests
+      // try {
```
Why are we skipping PRs?

buildgreatthings commented on packages/server/src/services/sync/adapters/github-adapter.ts:
```diff
@@ -0,0 +1,1144 @@
+      const owner = repo.owner.login;
+
+      const repoName = repo.name;
+      // // Fetch issues
```
Why are we skipping Issues?

buildgreatthings replied:
Looks like we sync it, we just don't index it.

buildgreatthings commented on packages/server/src/services/sync/adapters/github-adapter.ts:
```diff
@@ -0,0 +1,1144 @@
+      // }
+
+      // Fetch workflows
+      try {
```
Do we need workflows? These are mostly tests and deployments. Any ideas of how peopel might use this?

viveksingh101994 replied:
Yes, its majorly github action.

viveksingh101994 committed:
feat(github-adapter): re-enable GitHub issues and pull requests syncing

Uncomment code to fetch issues and pull requests from GitHub repositories.
This restores the ability to sync issues and PRs alongside workflows,
with proper error handling and batch processing for each resource type.

Previously, only workflows were being fetched while issues and PRs
functionality was commented out. This change enables full repository
data synchronization.

viveksingh101994 committed:
Merge branch 'github_mcp' of github.com:reality-platforms/ebee-oss into github_mcp

viveksingh101994 committed:
feat(client): add service icons and improve server card UI (#34)

- Add change-case dependency for proper text capitalization
- Implement dynamic service icons (Slack, GitHub, Fathom, Google Drive) based on server name
- Display server names using capital case formatting for better readability
- Add icon badges with purple theme to server cards
- Refactor MCPServerCard component to include getServiceIcon helper function

This enhances the visual identification of different MCP server types and improves overall UI consistency.

viveksingh101994 committed:
chore: removing google and adding notion
